SAMtags.tex

```diff
@@ -208,6 +212,16 @@ \subsection{Additional Template and Mapping data}
 \subsection{Metadata}
 
 \begin{description}
+
+\item[FT:Z:\tagvalue{text}] Filter field, free-text indicating the reason why this record has been modified or filtered
+There are two ways in which reads can be filtered: their 0x200 bit can be set (not passing filters) or their 0x4 bit can be set (segment unmapped). 
+In either case it may be important or interesting to inform the users that this action was done after the vendor created the record and/or the aligner had been run. 
+The {\tt FT} tag is a place to put such information. 
+For example, Picard's MergeBamAlignment will unmap (unset 0x4) a read (and its mate) if it finds that it had been clipped on both ends, and it has less than some number of bases that are \emph{not} clipped.
```

Comment:
I'm not sure including examples from specific tools in a spec document is a great idea.